db/seeds.rb

```diff
@@ -45,4 +71,22 @@ def clean_zipcode(zip_code)
     count += 1
     puts count
   end
+
+  # conditions.each do |row|
+  #   row=row.to_h
+  #   clean_row = {}
+  #   clean_row[:zip_code] = clean_zipcode(row[:zip_code])
+  #   clean_row[:max_temperature] = clean_conditions(row[:max_temperature_f])
+  #   clean_row[:mean_temperature] = clean_conditions(row[:mean_temperature_f])
+  #   clean_row[:min_temperature] = clean_conditions(row[:min_temperature_f])
+  #   clean_row[:mean_humidity] = clean_conditions(row[:mean_humidity])
+  #   clean_row[:mean_visibility] = clean_conditions(row[:mean_visibility_miles])
+  #   clean_row[:mean_wind_speed] = clean_conditions(row[:mean_wind_speed_mph])
+  #   clean_row[:precipitation] = clean_conditions(row[:precipitation_inches])
+  #   row[:weather_date] = row[:date]
+  #   clean_row[:weather_date] = seed_by_date(row[:weather_date])
+  #   puts "finished rows"
+  #   Condition.create!(clean_row) if pick_a_stupid_zipcode(row[:zip_code])
+  # end
```

Comment:
Is this just commented out because you're not seeding yet? Maybe throw this on a separate branch and wait to merge until you're ready to bring it in to master.